tests/Integration/Resources/Storage/Test_Resources_GFAL2StorageBase.py

```diff
@@ -262,7 +272,8 @@ def setUp( self ):
 
 if __name__ == '__main__':
   suite = unittest.defaultTestLoader.loadTestsFromTestCase( GFAL2_SRM2_Test )
-  suite.addTest( unittest.defaultTestLoader.loadTestsFromTestCase( GFAL2_XROOT_Test ) )
+  suite.addTest( unittest.defaultTestLoader.loadTestsFromTestCase( GFAL2_SRM2_Test ) )
```

Comment:
In any case, I am working on fixing this test. I will take your changes, but can you revert back all that not to create conflict please ?